### DISCUSSION TITLE
Beta: Updated Command Queue Handling

### DISCUSSION THREAD

**Discussion by tbnobody** [tbnobody/OpenDTU#2356] [General]
Would anyone like to test a optimised queue handling? This should eliminate dropouts when polling statistics while sending limit commands and prevent the queue from filling up. 

Before I commit this to the master branch I want to make sure that it doesn't break too many existing setups.
If there are interested people who have implemented a limit control (for all others it works anyway) and are willing to experiment (currently no guarantee on function) I could provide a `.patch` file. Alternatively a customised `generic_esp32.bin` (only this variant).

Another prerequisite would be that you have access to the serial console to provide debug output if something went wrong.

**Comment by stefan123t** [comment]
Would you mind creating a feature branch / PR so people can build it and/or review the code changes up front if you want some feedback on your Command Queue Habdling solution or is this still too premature ?

**Comment by tbnobody** [comment]
It's not to premature. It works very well for all my tests. I just don't want to test whether all github actions do what they are supposed to do right now.
